WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -352,6 +350,18 @@ public static void loginForDeeplink(Activity activity) {
         activity.startActivityForResult(intent, RequestCodes.DO_LOGIN);
     }
 
+    public static void loginForShareIntent(Activity activity) {
+        if (AppPrefs.isLoginWizardStyleActivated()) {
+            Intent intent = new Intent(activity, LoginActivity.class);
+            LoginMode.SHARE_INTENT.putInto(intent);
```

Comment:
I'm under the impression that this flow allows the magiclink login to happen, which by its nature breaks the continuation of the Sharing action after login. What do you think?

Reply:
Ohh, very good point! I haven't realized that.